build/templates/about_driver.inc.mako

```diff
@@ -0,0 +1,20 @@
+<%
+    import build.helper as helper
+
+    config        = template_parameters['metadata'].config
+    module_name   = config['module_name']
+    driver_name   = config['driver_name']
+%>\
+.. _about-section:
+
+About
+=====
+
+The **${module_name}** module provides a Python API for ${driver_name}. The code is maintained in the Open Source repository for `nimi-python <https://github.com/ni/nimi-python>`_.
+
```

Comment:
Do we have a place in which we say how to install the module, and the fact that the driver runtim must be installed separately? If not, a short `Installation` section here would be great.

Reply:
We already pull in the installation.inc for each module.
Example:

```
.. _nidcpower_installation-section:

Installation
------------

As a prerequisite to using the nidcpower module, you must install the NI-DCPower runtime on your system. Visit `ni.com/downloads <http://www.ni.com/downloads/>`_ to download the driver runtime for your devices.

The nimi-python modules (i.e. for **NI-DCPower**) can be installed with `pip <http://pypi.python.org/pypi/pip>`_::

  $ python -m pip install nidcpower
```
(That's not how it renders. To see that, just click on one of the generated READMEs in the PR.)